### PR TITLE
Introduce Entity/Relation Kinds as a first-class Objects

### PR DESCRIPTION
**Pull request by shadansari**

### CODE REVIEW

File: pkg/bulk/topo.go [hunk 1]
```diff
@@ -75,9 +94,11 @@ func GetTopoConfig(location string) (TopoConfig, error) {
 	if topoConfig == nil {
 		topoConfig = &TopoConfig{}
 		if err := configlib.LoadNamedConfig(location, topoConfig); err != nil {
+			fmt.Println("1##########")
```
Comment (SeanCondon) [inline pkg/bulk/topo.go:97]: might not be needed

File: pkg/bulk/topo.go [hunk 2]
```diff
@@ -25,10 +25,29 @@ var topoConfig *TopoConfig
 
 // TopoConfig - the top level object
 type TopoConfig struct {
+	TopoKinds     []TopoKind
 	TopoEntities  []TopoEntity
 	TopoRelations []TopoRelation
 }
 
+// TopoKind - required to get around the "onoof" Obj
+type TopoKind struct {
```
Comment (SeanCondon) [inline pkg/bulk/topo.go:34]: looking good

Comment (shadansari) [reply pkg/bulk/topo.go:34]: done

File: pkg/bulk/topo.go (hunk 1 above)
Comment (shadansari) [reply pkg/bulk/topo.go]: done

**Pull request merged**